apps/web/core/components/pages/auth/passcode/page-component.tsx

```
@@ -433,96 +433,104 @@ export function WorkSpaceComponent(props: IWorkSpace) {
+													) : (
+														<CircleIcon className="w-6 h-6" />
+													)}
+												</span>
+											</div>
+											<span
+												className={`bg-[#E5E5E5] w-full h-[1px] hidden ${expandedWorkspace === index && 'block'}`}
+											></span>
+											{/* <div className="w-full h-[1px] bg-[#E5E5E5] dark:bg-[#34353D]"></div> */}
+											<div className="flex flex-col gap-4 px-5 py-1.5">
+												{worksace.current_teams
+													?.filter((team) => team && team.team_name)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)